raster-foundry/raster-foundry
# Extend upload type enum in datamodel and database

## Overview

This PR adds the `MODIS_USGS` upload type to postgres enums and to the `datamodel`'s
`UploadType` enum.

### Checklist

- [x] [Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated (raster-foundry/raster-foundry-api-spec#9)

## Testing Instructions

 * run migrations
 * `SELECT unnest(enum_range(NULL::upload_type));` from postgres -- you should see `MODIS_USGS`

Closes #3238
